internal/store/postgres/migrations/000006_add_appeal_config_to_policies.up.sql

```diff
@@ -1,2 +1,2 @@
 ALTER TABLE "policies"
-    ADD "appeal_config" jsonb;
\ No newline at end of file
+    ADD "appeal_config" jsonb default '{"allow_cross_individual_user":false}';
```

Comment:
Can we do this changes without having a default json value like this?
Perhaps having a default value in the app logic itself?

Reply:
CMIIW but the domain.PolicyAppealConfig is a pointer in Policy; and if empty/default gets written to DB,  It will be `null`. It need not have this hard coded.

Reply:
done